lib/activerecord-import/import.rb

```diff
@@ -275,7 +275,11 @@ def import_helper( *args )
           # this next line breaks sqlite.so with a segmentation fault
           # if model.new_record? || options[:on_duplicate_key_update]
             column_names.map do |name|
-              model.read_attribute_before_type_cast(name.to_s)
+              if model.class.column_defaults[name.to_s].is_a? Integer
+                model.read_attribute(name.to_s)
+              else
+                model.read_attribute_before_type_cast(name.to_s)
```

Comment:
Can someone why `read_attribute_before_type_cast` is being called instead of just `read_attribute` anyway?